### ISSUE TITLE
Facepager keeps crashing on Mac High Sierra

### ISSUE THREAD

**Issue by SlickMonkey1990** [strohne/Facepager#84]
Facepager keeps crashing on Mac High Sierra. Can’t get it to launch...

**Comment by kathy223** [comment]
yep same problem. its the first time i tried to use facepager so i was coming here to see if anyone knew the problem with mac. as luck would have it, i just updated to Sierra a few hours ago. it's crashing on me as well, cannot open :(

**Comment by strohne** [comment]
Sad it doesn't work out of the box. Could you pleasy try out the latest build: https://github.com/strohne/Facepager/raw/master/build/osx/Facepager_3_9.app.zip

This is a much bigger file. But there are some more libraries included which maybe are missing on High Sierra.

**Comment by SlickMonkey1990** [comment]
That worked—thanks!

**Comment by kathy223** [comment]
I'll try it tonight I'm excited to hear it worked for slick 😁
Thanks,
Lynne

On Apr 3, 2018, at 1:01 PM, SlickMonkey1990 <notifications@github.com<mailto:notifications@github.com>> wrote:


That worked—thanks!
